src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -156,9 +164,9 @@ describe('Test suite for dictionary concepts components', () => {
       paginateConcepts: jest.fn(),
       totalConceptCount: 11,
     };
-    const wrapper = mount(<Router>
+    const wrapper = mount(<Provider store={store}><Router>
       <DictionaryConcepts {...props} />
-    </Router>);
+    </Router></Provider>);
     wrapper.find('#next').simulate('click');
     wrapper.find('#previous').simulate('click');
```

Comment:
This test is also not asserting anything. If it is not in your scope, please don't edit it.

Reply:
I have removed the edits